python-sdk/src/astro/files/locations/local.py

```diff
@@ -33,3 +33,19 @@ def size(self) -> int:
         """
         path = pathlib.Path(self.path)
         return os.path.getsize(path)
+
+    @property
+    def openlineage_dataset_namespace(self) -> str:
+        """
+        Returns the open lineage dataset namespace as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        """
+        return ""
```

Comment:
Why can we not have here  `parsed_url = urlparse(self.path)` like in other places as I see we're using this below to get dataset name.

Reply:
> Looks great start to me @sunank200 👏🏽 Should we add the local flask based open lineage server and a make target to it to test for local development like in the providers repository? What do you think?

@pankajkoti Its already added as part of this [PR](https://github.com/astronomer/astro-sdk/pull/1001) Its merged to main already

Reply:
> Why can we not have here `parsed_url = urlparse(self.path)` like in other places as I see we're using this below to get dataset name.

We can have that but this doc https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md doesn't specify the proper namespace for local file path

Reply:
I asked the open lineage team. They said in the case of the local file path, namespace can be file name and name can be filepath